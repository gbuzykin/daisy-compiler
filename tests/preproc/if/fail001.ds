#define SUM(a, b) ((a) + (b))
#if SUM(3, 6) > SUM(2u, 5u)
#if SUM(3, 6) > SUM(3, 6) / -SUM(0, 0)
#endif
#endif
