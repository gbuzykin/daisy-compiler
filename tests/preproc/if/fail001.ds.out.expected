./preproc/if/fail001.ds:2:15: warning: signed/unsigned mismatch
 2 | #if SUM(3, 6) > SUM(2u, 5u)
   |               ^
./preproc/if/fail001.ds:3:29: error: integer division by zero
 3 | #if SUM(3, 6) > SUM(3, 6) / -SUM(0, 0)
   |                             ^~~~~~~~~~
./preproc/if/fail001.ds: info: warnings 1, errors 1
